osu.Game/Screens/Play/ModsContainer.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics.Containers;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.UI;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.Sprites;
+using OpenTK;
+
+namespace osu.Game.Screens.Play
+{
+    public class ModsContainer : Container
+    {
+        private readonly FillFlowContainer<ModIcon> iconsContainer;
+
+        private bool showMods;
+        public bool ShowMods
+        {
+            set
+            {
+                showMods = value;
+                if (!showMods) Hide();
+            }
+            get { return showMods; }

```

Comment:
get comes before set. also match braces style (put content on newline if content in set is on newline).